libs/defi/shared/src/components/TxButton/hooks/useTxButton.ts

```diff
@@ -206,37 +179,35 @@ export const useTxButton = <
   const onSimulateSuccess = useCallback(
     (data: SimulateContractReturnType) => {
       setSimulateError(undefined);
-      if (args.enableGas && isConnected) {
+      if (args?.enableGas && isConnected) {
         refetchGas();
       }
-      args.callbacks?.onSimulateSuccess?.(data);
+      args?.callbacks?.onSimulateSuccess?.(data);
```

Comment:
im confused why all these changed -- am I wrong? can `args` be undefined?

Reply:
leftovers from another tryout, updated